Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/Collections/GridObjectCollection.cs

```diff
@@ -149,8 +175,8 @@ public Mesh CylinderMesh
         #region private fields
 
         private int columns;
-
-        private float circumference;
+        
+        //private float circumference;
```

Comment:
If it's not needed, then simply remove it (if you need it later, GIT will remember :D)